lib/app/widgets/menu.ex

```diff
@@ -0,0 +1,50 @@
+defmodule App.Widgets.Menu do
+  use Widgets
+
+  @name "menu"
+  def menu_w(_data, _props, _context) do
+    %{
+      "child" => %{
+        "children" => [
+          %{
+            "child" => %{"src" => "logo.png", "type" => "image"},
+            "constraints" => %{
+              "maxHeight" => 32,
+              "maxWidth" => 32,
+              "minHeight" => 32,
+              "minWidth" => 32
+            },
+            "type" => "container"
+          },
+          %{
+            "child" => %{
+              "child" => %{
+                "style" => %{"fontSize" => 24, "fontWeight" => "bold"},
+                "textAlign" => "center",
+                "type" => "text",
+                "value" => "Hello World"
+              },
+              "type" => "container"
+            },
+            "type" => "flexible"
+          }
+        ],
+        "crossAxisAlignment" => "center",
+        "fillParent" => true,
+        "mainAxisAlignment" => "spaceBetween",
+        "padding" => %{"right" => 4},
+        "type" => "flex"
+      },
+      "decoration" => %{
+        "boxShadow" => %{
+          "blurRadius" => 8,
+          "color" => 436_207_616,
+          "offset" => %{"dx" => 0, "dy" => 1}
+        },
+        "color" => 4_294_967_295
```

Comment:
Is there an hexadecimal int form in Elixir to make it easier ?

Reply:
probably, i've just copy/paste the node version, my bad :')